### PR TITLE
Adding python-for-android recipe

### PR DESCRIPTION
**Pull request by EntPyle**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-for-android/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - python2
+    - cython
+    - openjdk-8
+    - zlib
+    - libncurses
+    - unzip
+    - virtualenv
+    - autoconf
+    - libtool
+    - cmake
```
Comment (marcelotrevisani) [inline recipes/python-for-android/meta.yaml]: These imports should be just this project ``python-for-android``

File: recipes/python-for-android/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 701bf784958a958d70222a6d530f5cf2b44603e183638b35fa2a3de6476afba4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (marcelotrevisani) [inline recipes/python-for-android/meta.yaml]: Do you need the ``C`` compiler?  It seems that you are not compiling anything, am I right?

Comment (marcelotrevisani) [reply recipes/python-for-android/meta.yaml]: I think you can delete this part

File: recipes/python-for-android/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 701bf784958a958d70222a6d530f5cf2b44603e183638b35fa2a3de6476afba4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/python-for-android/meta.yaml]: If you do not need the compiler please add this recipe as ``noarch: python``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

**Comment by EntPyle** [comment]
According to the pypi page, the module contains some C files, so I assumed I’d need the compiler. I’m happy to update the test module listing. It is still OS independent so I think noarch will still work.

File: recipes/python-for-android/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/python-for-android/meta.yaml]: Yeah, these seem like run deps

File: recipes/python-for-android/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{% set name = "python-for-android" %}
+  {% set version = "2019.7.8" %}
```
Comment (scopatz) [inline recipes/python-for-android/meta.yaml]: This shouldn't be indented

File: recipes/python-for-android/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - python-for-android
```
Comment (scopatz) [inline recipes/python-for-android/meta.yaml]: Python modules don't use dashes so this can't be right.

File: recipes/python-for-android/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+about:
+  home: https://github.com/kivy/python-for-android
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Turn your Python application into an Android APK '
```
Comment (scopatz) [inline recipes/python-for-android/meta.yaml]: extra space at the end of the string

**Pull request closed without merging**